docs-ref-services/iot-hub.md

```diff
@@ -24,17 +24,80 @@ Azure IoT Hub is a fully managed service that enables reliable and secure bidire
 - Provides extensive monitoring for device connectivity and device identity management events.
 - Includes device libraries for the most popular languages and platforms.
 
-## Install the modules with npm
-
 Use npm to install the Azure IoT Hub modules for Node.js
 
-### Management
+## Management Package
+
+### Install the npm packages
 ```bash
 npm install azure-arm-iothub
 ```
 
-##Example
+### Example
+
+```javascript
+const msRestAzure = require('ms-rest-azure');
+const IoTHubClient = require('azure-arm-iothub');
+
+const subscriptionId = 'your-subscription-id';
+const resourceGroup = 'testrg001';
+const hubName = 'testhub';
+const location = 'East US';
+
+// Describe the IoT hub you want to create
+const hubDescription = {
+  name: hubName,
+  location: location,
+  subscriptionid: subscriptionId,
+  resourcegroup: resourceGroup,
+  sku: { name: 'S1', capacity: 2 },
+  properties: {
+    enableFileUploadNotifications: false,
+    ipFilterRules: [{ filterName: 'ipfilterrule', action: 'accept', ipMask: '0.0.0.0/0' }],
+    operationsMonitoringProperties: {
+      events: {
+        C2DCommands: 'Error',
+        DeviceTelemetry: 'Error',
+        DeviceIdentityOperations: 'Error',
+        Connections: 'Error, Information'
+      }
+    },
+    features: 'None'
+  }
+};
+
+// Interactive Login
+msRestAzure.interactiveLogin((err, credentials) => {
+  const client = new IoTHubClient(credentials, subscriptionId);
+
+  client.iotHubResource
+    .createOrUpdate(resourceGroup, hubName, hubDescription)
+    .then(hubDescriptionResult => console.log(hubDescriptionResult))
+    .catch(err => console.error(err));
+});
+```
+
+Get the existing hub, by name
+
+```javascript
+const subscriptionId = 'your-subscription-id';
+const resourceGroup = 'testrg001';
+const hubName = 'testhub';

```

Comment:
I would change these to be consistent that this requires _their_ resource group and hub name. `your-resource-group`, `your-hub-name`